app/models/hesa/trainee.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Hesa
+  class Trainee < ApplicationRecord
+    self.table_name = "hesa_trainees"
+    belongs_to :trainee,
```

Comment:
Do we need to add the relationship? I suppose it's convenient but we're integrating what will hopefully be temporary debugging data into the app. Probably not a 'biggy'. Also should one of these be a has_one if we do?